src/components/ConnectionRate.vue

```diff
@@ -21,9 +21,18 @@
       call.value.on(
         VoxImplant.CallEvents.CallStatsReceived,
         (e: EventHandlers.CallStatsReceived) => {
-          if (e.stats.totalLoss && e.stats.totalLoss <= 0.01) {
+          const totalPacketsReceived = Object.values(e.stats.inbound).reduce((acc, inbound) => {
+            acc += inbound.packetsReceived;
+            return acc;
+          }, 0);
+          const totalPacketsLost = Object.values(e.stats.inbound).reduce((acc, inbound) => {
+            acc += inbound.packetsLost;
+            return acc;
+          }, 0);
+          const lossRate = totalPacketsReceived ? totalPacketsLost / totalPacketsReceived : 0;
```

Comment:
the correct formula for packet loss is

totalPacketsLost / (totalPacketsLost + totalPacketsReceived)